### PR TITLE
Added more content

### PR DESCRIPTION
**Pull request by arjunpandey2003**
Issue #39

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -27,4 +31,4 @@ FOSSASIA Organization Team
 ####Hong Phuc Dang
 FOSSASIA Organizer
 
-[hp@fossasia.org](mailto:hp@fossasia.org)
\ No newline at end of file
+[hp@fossasia.org](mailto:hp@fossasia.org)
```
Comment (0x48piraj) [inline about.md]: Don't delete and replace same stuff, if possible.

**Review by 0x48piraj — APPROVED** [review]
LGTM! :+1:

File: about.md [hunk 2]
```diff
@@ -10,12 +10,16 @@ FOSSASIA 2014 is an exciting Open Technology event for developers, students, and
 
 Since the first FOSSASIA meetup in 2009 in Ho Chi Minh City, FOSSASIA has taken the lead to organize Free and Open Source Software conferences in Vietnam. Past FOSSASIA events have attracted over 1400 participants and speakers from 24 countries, with more than 100 volunteers. More than half of all participants were women.
 
+FOSSASIA is an organization developing software applications for social change using a wide-range of technologies. It was established 2009. Projects range from Free and Open Source software, to design, graphics and hardware. We also organize and participate in conferences, meetups and code camps. The annual FOSSASIA OpenTechSummit is one of the largest gatherings of developers in Asia and took place in Vietnam, Cambodia and Singapore. Please join us and start contributing to our projects on [GitHub](https://github.com/fossasi.org).
```
Comment (gabru-md) [inline about.md]: It was established `in` 2009.

`in` is missing

File: about.md [hunk 3]
```diff
@@ -10,12 +10,16 @@ FOSSASIA 2014 is an exciting Open Technology event for developers, students, and
...
+
 ##FOSSASIA Activities
 
 FOSSASIA works together with Open Source projects in Asia and around the world and develops Free and Open Source software for social change.
 
 We bring developers, designers and start ups together, provide infrastructure and organize code sprints, developer meet ups and community gatherings.
 
+Every year Fossasia participates in events like [Google Code](https://codein.withgoogle.com) in and [Google summer of Code](https://summerofcode.withgoogle.com/), allowing young developers to contribute to open-source. Each year Fossasia becomes the recipient of the most participants, and collectively they do 100s of tasks, helping Fossasia and at the same time improving as developers.
```
Comment (gabru-md) [inline about.md]: Google Code In is a single entity and `[Google Code]` must be replaced by `[Google Code-In]` and delete the extra `in`

File: about.md (hunk 1 above)
Comment (gabru-md) [reply about.md]: These changes are generally non-intentional and occur due to extra spacing. So It is okay! :)

**Review by gabru-md — CHANGES_REQUESTED** [review]
Please make the necessary changes

**Comment by arjunpandey2003** [comment]
@gabru-md and @0x48piraj  updated

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
squash the commis

File: about.md [hunk 4]
```diff
@@ -10,15 +10,15 @@ FOSSASIA 2014 is an exciting Open Technology event for developers, students, and
...
 
 ##FOSSASIA Activities
 
 FOSSASIA works together with Open Source projects in Asia and around the world and develops Free and Open Source software for social change.
 
 We bring developers, designers and start ups together, provide infrastructure and organize code sprints, developer meet ups and community gatherings.
 
-Every year Fossasia participates in events like [Google Code](https://codein.withgoogle.com) in and [Google summer of Code](https://summerofcode.withgoogle.com/), allowing young developers to contribute to open-source. Each year Fossasia becomes the recipient of the most participants, and collectively they do 100s of tasks, helping Fossasia and at the same time improving as developers.
+Every year Fossasia participates in events like [Google Code In](https://codein.withgoogle.com) in and [Google summer of Code](https://summerofcode.withgoogle.com/), allowing young developers to contribute to open-source. Each year Fossasia becomes the recipient of the most participants, and collectively they do 100s of tasks, helping Fossasia and at the same time improving as developers.
```
Comment (gabru-md) [inline about.md]: Delete the `in` after `Google Code In`
and change `Google Code In` to ` Google Code-In`

**Review by gabru-md — CHANGES_REQUESTED** [review]

**Comment by arjunpandey2003** [comment]
@0x48piraj and @gabru-md made the changes

**Comment by gabru-md** [comment]
Your PR has files changed can you please look into it? There is a 404.html file added as well which is not mentioned anywhere in the issue #39

**Comment by gabru-md** [comment]
Please Add the markdown changes that you made.
Now those changes are no longer available you have deleted them as well.

**Comment by arjunpandey2003** [comment]
Yeah i am making the changes again you can close this request

**Comment by gabru-md** [comment]
Closing this Pull Request

**Pull request closed without merging**